.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__/
+env/
+*.db
```

Comment:
Addign gitignore file is really helpful so git does not always ask you if you need to push these files that should never be pushed for example like the cache